tutorial/Lab0/README.md

```diff
@@ -1,26 +1,20 @@
 # Lab 0. Installing Helm on IBM Cloud Kubernetes Service
 
-There are two parts to installing Helm: the client (helm) and the server (Tiller). Helm can be installed from source or pre-built binary releases. In this lab, we are going to use the pre-built binary release (Linux amd64) from the Helm community. Refer to the [Helm install docs](https://v2.helm.sh/docs/using_helm/#install-helm) for more details. 
+The Helm client (`helm`) can be installed from source or pre-built binary releases. In this lab, we are going to use the pre-built binary release (Linux amd64) from the Helm community. Refer to the [Helm install docs](https://helm.sh/docs/intro/install/) for more details. 
 
 # Prerequisites
 
 Create a Kubernetes cluster with [IBM Cloud Kubernetes Service](https://cloud.ibm.com/docs/containers/cs_tutorials.html#cs_cluster_tutorial), following the steps to also configure the IBM Cloud CLI with the Kubernetes Service plug-in.
 
 # Installing the Helm Client (helm)
 
-1. Download the [latest release Helm](https://github.com/helm/helm/releases) for your environment, which in this case is `Linux amd64`.
+1. Download the [latest release of Helm v3](https://github.com/helm/helm/releases) for your environment, which in this case is `Linux amd64`.
```

Comment:
Suggest to use "latest release of Helm" (take out v3)

Reply:
Unfortunately at the moment, the releases in GitHub will show in order when a releases is published. This mean that you could have a v2 releases showing before a v3 releases. This is why I am specifying v3 here.